scripts/pages/pgSettings.ts

```diff
@@ -0,0 +1,91 @@
+import PgSettingsDesign from 'generated/pages/pgSettings';
+import Data from "sf-core/global/data";
+import Application from 'sf-core/application';
+import AttributedString from 'sf-core/ui/attributedstring';
+import Menu from 'sf-core/ui/menu';
+import MenuItem from 'sf-core/ui/menuitem';
+import AlertView from 'sf-core/ui/alertview';
+import { ThemeService } from 'theme';
+import { setLanguage } from 'lib/language-helper';
+
+export default class PgSettings extends PgSettingsDesign {
+    router: any;
+    currentTheme: string;
+    constructor() {
+        super();
+        // Overrides super.onShow method
+        this.onShow = onShow.bind(this, this.onShow.bind(this));
+        // Overrides super.onLoad method
+        this.onLoad = onLoad.bind(this, this.onLoad.bind(this));
+
+        this.currentTheme = Data.getStringVariable('currentTheme');
```

Comment:
You could have wrapped this method inside of theme.ts for better readability. But this is also fine.